migrations/sqls/20191113023354-remove-youth-groups-leadership-dev-up.sql

```diff
@@ -0,0 +1,11 @@
+SELECT @oldID := id FROM services WHERE serviceName = 'Youth Groups / Leadership Development';
+SELECT @newID := id FROM services WHERE serviceName = 'Youth Groups';
+
+UPDATE
+  `intakeServices`
```

Comment:
We also need to update the `leadServices` and `clientInteractions` tables in a similar fashion

Reply:
🙀

Reply:
updated